packages/synapse-react-client/src/components/SynapseForm/SynapseFormSubmissionGrid.tsx

```diff
@@ -518,4 +519,13 @@ export class SynapseFormSubmissionGrid extends React.Component<
   }
 }
 
+export function SynapseFormSubmissionGrid(
```

Comment:
Today, our portals RouteResolver injects a `token` prop into every component, which is a vestige of how we used to path the auth token using prop drilling. 

A few of the SynapseForm class components still rely on this, and I want to get rid of RouteResolver, so I wrapped the existing component with a function component, where I could pull the token out of a `useContext` hook.